controllers/user.server.controller.js

```diff
@@ -124,3 +124,35 @@ exports.userUpdateById = function(req, res) {
         res.status(400).send("Bad request");
     }
 }
+
+/**
+ * Delete the data of an existing forum user matching a given ID using HTTP request object data
+ * @param req HTTP request object
+ * @param res HTTP request response object
+ */
+ exports.userDeleteById = function(req, res) {
+    const reqParams = req.params;
+
+    // Check that a valid ID exists in the request parameters
+    let isBadRequest = !isValidDocumentID(reqParams.id);
+
+    if (!isBadRequest) {
```

Comment:
```suggestion
       if (isValidDocumentID(reqParams.id)) {
```
It seems redundant to still use isBadRequest